### PR TITLE
Update the EKS Add-on advanced configuration docs

### PR DESCRIPTION
**Pull request by humivo**
Make updates to the EKS Add-on advanced configuration docs for the breaking change in v0.88.0.  Also, restructure and reformat the EKS Add-on docs in general for readability and for a better user experience.

### CODE REVIEW

**Review by bryan-aguilar — APPROVED** [review]

File: src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+| `collector.xray.enabled`                    | Specify whether or not to enable X-Ray as a destination for Collector deployment.                                                                          | `false`              | "{"collector":{"xray":{"enabled":true}}}"                                                                                                                               |
+
+*Note that in Fargate, resource requests and limits must be equal, see this [troubleshooting guide](https://docs.aws.amazon.com/eks/latest/userguide/troubleshooting-adot.html) for more information.
+
+**This configuration value can be used to annotate your service account and associate it with an IAM role for the IAM Roles for Service Accounts (IRSA). As an alternative to using this configuration value for IRSA, you can use the `--service-account-role-arn` flag when creating or updating the add-on. This flag will annotate service accounts created by the add-on with the role ARN you provide. The equivalent to the example provided for `collector.serviceAccount.annotations` is `--service-account-role-arn arn:aws:iam::000000000000:role/adot-collector`.
```
Comment (mhausenblas) [inline src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx:32]: What is "This" in This configuration value?

File: src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+title: 'EKS add-ons Advanced Configuration for ADOT: Collector Deployment (<v0.88.0-eksbuild.1)'
+description:
+    This page details Collector deployment through EKS add-on advanced configuration for ADOT.
+path: '/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated'
+---
+
+As discussed in the section [Add-on Advanced Configuration (&lt;v0.88.0-eksbuild.1)](/docs/getting-started/adot-eks-add-on/add-on-configuration-deprecated), EKS add-ons now provides the ability to configure ADOT during installation time of the add-on. With this functionality, an ADOT Collector can also be deployed during an installation, provided that add-on version `v0.62.1-eksbuild.1` or higher is being used.
+
+For more information on Collector configuration, and to learn about Amazon Managed Service for Prometheus (AMP), AWS CloudWatch (CW), and AWS X-Ray as telemetry destinations, see the [Collector configuration introduction](/docs/getting-started/adot-eks-add-on/config-collector-intro) section.
```
[the remark this answers is not in the document]
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx]: Changed to Amazon CloudWatch

File: src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx (hunk 1 above)
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx]: Specified "this"

File: src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Use your IAM role to launch the ADOT Collector
+You can associate your IAM role to your EKS service account using [IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html). Your service account can then provide AWS permissions to the containers you run in any pod that use that service account. You must use this command for each cluster where you're installing ADOT to grant your AWS service account permissions.
+Follow these steps to associate your IAM role to your EKS service account using IRSA:
+
+1. Create an IAM OIDC provider for your cluster by following the steps in the link below:
```
[the remark this answers is not in the document]
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment-deprecated.mdx]: Spelled out OIDC now.

File: src/docs/getting-started/adot-eks-add-on/add-on-configuration-deprecated.mdx [hunk 4]
```diff
@@ -0,0 +1,98 @@
+---
+title: 'EKS add-ons Advanced Configuration for ADOT (<v0.88.0-eksbuild.1)'
+description:
+    This page introduces EKS add-on advanced configuration for ADOT.
+path: '/docs/getting-started/adot-eks-add-on/add-on-configuration-deprecated'
+---
+
+> **IMPORTANT**: The following documentation is for ADOT EKS-Addon versions preceeding `v0.88.0-eksbuild.1`.  If you are using a version greater than or equal to `v0.88.0-eksbuild.1`,
```
[the remark this answers is not in the document]
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/add-on-configuration-deprecated.mdx]: Changed to "less than"

File: src/docs/getting-started/adot-eks-add-on/config-otlp-ingest.mdx [hunk 5]
```diff
@@ -0,0 +1,103 @@
+---
+title: 'OTLP Ingest Collector Configuration'
+description:
+    This page introduces Collector configuration for OTLP Ingest for the ADOT Collector.
+path: '/docs/getting-started/adot-eks-add-on/config-otlp-ingest'
+---
+The OTLP Ingest collector configuration launches a preconfigured OpenTelemetryCollector custom resource
```
[the remark this answers is not in the document]
Comment (humivo) [reply src/docs/getting-started/adot-eks-add-on/config-otlp-ingest.mdx]: Expanded OTLP and linked to docs

**Review by mhausenblas — APPROVED** [review]

**Pull request merged**